Makefile

```diff
@@ -16,8 +16,22 @@
 verify-boilerplate:
 	@./hack/verify-boilerplate.py --rootdir=$(CURDIR) --boilerplate-dir=hack/boilerplate && echo "Boilerplate verification passed."
 
+.PHONY: prowgen
+prowgen:
+	cd ./config/prowgen/ && go run . --branch=* -o $(CURDIR)/config/jobs/cert-manager/cert-manager/
+
+.PHONY: verify-prowgen
+verify-prowgen:
+	mkdir -p _temp
+	cd ./config/prowgen/ && go run . --branch=* -o $(CURDIR)/_temp/cert-manager/
+	diff -q -r $(CURDIR)/config/jobs/cert-manager/cert-manager/ $(CURDIR)/_temp/cert-manager/
```

Comment:
@inteon I am not sure about this - for example if we run the check now it will require that the changes such as the CPU request bump are applied to release-1.10 tests where they are not actually needed. I think before we can have a check like this any changes should be configurable for a specific branch only